build.sbt

```diff
@@ -1,3 +1,4 @@
+import ProjectPlugin.autoImport.micrositeSettings
```

Comment:
This wouldn't be necessary either

Reply:
Removing this import made an ambigous reference to `micrositeSettings`. 
```
it is imported twice in the same scope by
import ProjectPlugin.autoImport._
and import _root_.freestyle.FreestylePlugin.autoImport._
  .settings(micrositeSettings: _*)
```
To solve this I renamed the value as `freesMicrositeSettings`